packages/driver/src/cy/commands/navigation.ts

```diff
@@ -50,32 +50,32 @@ const isValidVisitMethod = (method) => {
 const timedOutWaitingForPageLoad = (ms, log) => {
   debug('timedOutWaitingForPageLoad')
 
-  const anticipatedCrossOriginHref = cy.state('anticipatingMultiDomain')
+  const anticipatedCrossOriginHref = cy.state('anticipatingCrossOriginRequest')
```

Comment:
Would it be more appropriate to use `Response` instead of `Request` since we delay the response not the request?

Reply:
should we include in the name that we are anticipating the response to be cross origin html?

Reply:
You're right, we are anticipating the response, the confusing thing is that we store the request in it. let me see what i can do.

Reply:
I updated the name and stored the request.. that seems more clear??? the request pairedd with the anticipated response???